kbs/tools/client/Cargo.toml

```diff
@@ -18,7 +18,7 @@ base64.workspace = true
 clap = { version = "4.0.29", features = ["derive"] }
 env_logger.workspace = true
 jwt-simple = "0.11.4"
-kbs_protocol = { git = "https://github.com/confidential-containers/guest-components.git", rev = "21b2c536b4d6c5c1442b53916c908b54dde136e8", default-features = false }
+kbs_protocol = { git = "https://github.com/confidential-containers/guest-components", rev="5ae8d8ed136aef6303dc9802df04d19b86e1e70d", default-features = false }
```

Comment:
```suggestion
kbs_protocol = { git = "https://github.com/confidential-containers/guest-components.git", rev="5ae8d8ed136aef6303dc9802df04d19b86e1e70d", default-features = false }
```

Reply:
Sorry for this.. amended the commit to add it.